### PR TITLE
Document the embedded Service LB for K3s

### PR DESCRIPTION
**Pull request by catherineluse**
https://github.com/rancher/k3s/issues/185

### CODE REVIEW

File: content/k3s/latest/en/networking/_index.md [hunk 1]
```diff
@@ -3,41 +3,80 @@ title: "Networking"
 weight: 35
 ---
 
->**Note:** CNI options are covered in detail on the [Installation Network Options]({{<baseurl>}}/k3s/latest/en/installation/network-options/) page. Please reference that page for details on Flannel and the various flannel backend options or how to set up your own CNI.
+This page explains how CoreDNS, the Traefik Ingress controller, and the built-in service load balancer work with K3s.
```
Comment (brandond) [inline content/k3s/latest/en/networking/_index.md:6]:
```suggestion
This page explains how CoreDNS, the Traefik Ingress controller, and Klipper service load balancer work within K3s.
```

File: content/k3s/latest/en/networking/_index.md [hunk 2]
```diff
@@ -3,41 +3,80 @@ title: "Networking"
...
 
-Open Ports
-----------
-Please reference the [Installation Requirements]({{<baseurl>}}/k3s/latest/en/installation/installation-requirements/#networking) page for port information.
+For information on CNI options, refer to the [Installation Network Options]({{<baseurl>}}/k3s/latest/en/installation/network-options/) page. Please reference that page for details on Flannel and the various flannel backend options or how to set up your own CNI.
```
Comment (brandond) [inline content/k3s/latest/en/networking/_index.md:8]:
```suggestion
Refer to the [Installation Network Options]({{<baseurl>}}/k3s/latest/en/installation/network-options/) page for details on Flannel configuration options and backend selection, or how to set up your own CNI.
```

File: content/k3s/latest/en/networking/_index.md [hunk 3]
```diff
@@ -3,41 +3,80 @@ title: "Networking"
...
-You can tweak traefik to meet your needs by setting options in the traefik.yaml file. Refer to the official [Traefik for Helm Configuration Parameters](https://github.com/helm/charts/tree/master/stable/traefik#configuration) readme for more information.
+Traefik can be configured by editing the `traefik.yaml` file. For more information, refer to the official [Traefik for Helm Configuration Parameters.](https://github.com/helm/charts/tree/master/stable/traefik#configuration)
 
 To disable it, start each server with the `--disable traefik` option.
 
-Service Load Balancer
----------------------
+# Service Load Balancer
+
+Any service load balancer (LB) can be leveraged in your Kubernetes cluster. K3s provides a Service LB that uses available host ports.
```
Comment (brandond) [inline content/k3s/latest/en/networking/_index.md:40]:
```suggestion
Any service load balancer (LB) can be leveraged in your Kubernetes cluster. K3s provides a load balancer known as Klipper that uses available host ports.
```

**Comment by catherineluse** [comment]
The changes look good to me.

**Pull request merged**